config/crd/bases/ref.nephio.org_configref.yaml

```diff
@@ -0,0 +1,60 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: configref.ref.nephio.org
```

Comment:
would it not be better to make this config iso configref since we are already in a ref namespace.

Reply:
Done